data-raw/fas_example_data.R

```diff
@@ -0,0 +1,181 @@
+  pivot_longer(
+    c(
+      matches('^Percent of'),
+      matches('^Number')
+    ),
+    names_to = 'name',
+    values_to = 'value'
+  ) %>%
+  mutate(value_type = case_when(
+    grepl('^Percent', name) ~ 'percent',
+    grepl('^Number', name) ~ 'count',
+  )) %>%
+  mutate(
+    cell_population = name,
+    cell_population = sub('Number of ', '', cell_population),
+    # for a typo in original dataset
+    cell_population = sub('Number ', '', cell_population),
+    cell_population = sub('Percent of .+ (that are|detected as) ', '', cell_population)
+  ) %>%
+  mutate(
+    percent_denominator = if_else(
+      value_type == 'percent',
+      gsub('^Percent of | ((that are)|(detected as)) .+$', '', name),
+      NA_character_
+    ),
+    percent_denominator = case_match(
+      percent_denominator,
+      'GT8++IgG+ B cells' ~ 'GT8++ IgG+ B cells',
+      "epitope-specific (KO-GT8++) sequenced IgG BCRs" ~ "sequenced epitope-specific (KO-GT8++) IgG+ B cells",
+      .default = percent_denominator
+    ),
+    cell_population = case_match(
+      cell_population,
+      "epitope-specific (KO-GT8++) sequenced IgG BCRs that are VRC01-class" ~ 'VRC01-class',
+      "KO-" ~ 'GT8++KO-',
+      "IgD-IgG+ B cells" ~ 'IgG+',
+      "IgD-IgG+ B cells that are GT8++ (without regard to KO binding status)" ~ 'GT8++ IgG+',
+      "epitope-specific (KO-GT8++) IgG+ B cells that have BCR heavy and light chains sequenced" ~ "sequenced epitope-specific (KO-GT8++) IgG+ B cells",
+      .default = gsub(' \\(without regard to KO binding status\\)|IgD\\-', '', cell_population)
+    ),
+    bcell_population = cell_population,
+    igx_type = if_else(grepl('IgG', name), 'IgG+', NA_character_),
+    Group = NA_character_,
+    dose = NA_character_,
+    dose_unit = NA_character_,
+    visitno = sub('^V', '', Visit),
+    visit = NA_character_,
+    visitunits = NA_character_,
+    antigen_specificity = if_else(grepl('GT8[+][+]|VRC01[-]class', name), 'GT8++', NA_character_),
+    epitope_specificity = if_else(grepl('KO[-]|VRC01[-]class', name), 'KO-', NA_character_),
+    bnab_class = if_else(grepl('^VRC01[-]class$', cell_population), 'VRC01-class', NA_character_),
+    source_assay = if_else(grepl('VRC01[-]class|sequenced', name), 'sequencing', 'flow'),
+    flag_bound = NA,
+    probeset = NA,
+    sample_type = 'PBMC',
+    source_file = 'https://github.com/SchiefLab/G001/raw/main/data/figures/flow_summary/flow_and_sequences.csv.gz',
+    PubID = sub('^PubID_', '', PubID)
+  ) %>%
+  # column cleanup
+  select(
+    -cell_population, -Visit
+  ) %>%
+  # Select/rename/reorder columns
+  select(
+    PubID,
+    Group,
+    Treatment,
+    dose,
+    dose_unit,
+    visitno,
+    visit,
+    visitunits,
+    Week = weeks_post,
+    bcell_population,
+    igx_type,
+    antigen_specificity,
+    epitope_specificity,
+    bnab_class,
+    source_assay,
+    flag_bound,
+    probeset,
+    sample_type,
+    source_file,
+    name,
```

Comment:
```suggestion
    endpoint_name = name,
```

I think this is a clearer column name